### PR TITLE
fixed

### LINKED ISSUE #4: Login pop-up
**As a Registered User, I want to be able to log in with credentials/Google account, so that I can access the personal cabinet in a Tutor/Student role**

_Precondition:_ 
The User is registered as a Student/Tutor.

_Acceptance criteria:_
1.	When a User clicks the “Login” button on the home page header, then the Login pop-up appears. 
2.	The Login pop-up has three adaptations: for desktop/laptop version, for tablet version, for mobile version. The Login pop-up in desktop/laptop version contains two parts: 
- static visualization (image) on the left;
- the registration block on the right.
The Mockup:
Image

The Login pop-up in tablet and mobile versions contains the registration block only.
The Mockup:
Image

3.	The pop-up has the   close button in the upper right corner.
4.	When a User clicks the   close button, the Login pop-up closes and a Guest is redirected to the last viewed page.
5.	When a User clicks outside the Login pop-up field, nothing happens.

### PR DESCRIPTION
**Pull request by markgol777**

### CODE REVIEW

**Review by mistercapxx — APPROVED** [review]

**Review by CallMeMaverick — APPROVED** [review]

**Review by Bakalina — APPROVED** [review]

**Review by Yaroslav-Yakovlev — APPROVED** [review]

File: src/containers/guest-home-page/login-form/LoginForm.jsx [hunk 1]
```diff
@@ -26,12 +26,18 @@ const LoginForm = ({
 
   const { openModal } = useModalContext()
 
+  // const { closeModal } = useModalContext();
+
```
Comment (nrevutska) [inline src/containers/guest-home-page/login-form/LoginForm.jsx]: we cannot leave these comments in the code

File: tsconfig.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx",
+    "jsx": "preserve",
```
Comment (nrevutska) [inline tsconfig.json:17]: why this change was made and how is it related to the task?

**Review by nrevutska — CHANGES_REQUESTED** [review]
the task linked to this pr does not correspond to the changes that were made

File: src/containers/guest-home-page/login-form/LoginForm.jsx (hunk 1 above)
Comment (markgol777) [reply src/containers/guest-home-page/login-form/LoginForm.jsx]: sorry forgot to remove

File: tsconfig.json (hunk 2 above)
Comment (markgol777) [reply tsconfig.json:17]: I kept getting compiling errors so I was forced to change it

**Review by nrevutska — APPROVED** [review]

**Pull request merged**